components/TaskCard/TaskCard.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Children } from 'react'
+import styles from './TaskCard.module.css'
+import Title from '../Title/Title'
+import Paragraph from '../Paragraph/Paragraph'
+import ButtonContainer from '../ButtonContainer/ButtonContainer'
+import Button from '../Button/Button'
+
+export default function TaskCard({...props}) {
+  return (
+    <div className={styles.taskCard}>
+      <Title>{props.title}</Title>
+      <Paragraph>Estimated due date: {props.date}</Paragraph>
```

Comment:
One cool thing you can do is display the "relative" time here.   

So instead of displaying 2023-03-01   
Displaying: 3 months from now.  

You can write this logic yourself, but I can recommend the excellent [date-fns](https://date-fns.org/v2.29.3/docs/formatDistance) package and the formatDistance function